docker-compose.base.yml

```diff
@@ -160,6 +160,8 @@ services:
             args:
                 BIN: capture
         restart: on-failure
+        ports:
+            - 3000:3000
```

Comment:
This was supposed to quickly solve a lot of pointless errors in logs (this is what I asked Frank to review here), but apparently there's already a PR (https://github.com/PostHog/posthog/pull/27008) and a major discussion around proper routing of capture has ensued ([Slack](https://posthog.slack.com/archives/C0113360FFV/p1734696143793739)) – so reverted here